src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java

```diff
@@ -177,40 +195,6 @@ public void shouldTransformAddresses() {
         );
     }
 
-    @Test
-    public void shouldMapPhoneNumber() {
```

Comment:
why have you removed these?

Reply:
sorry, pasted to the wrong window 🙈  
they were moved to shouldPassValidationForValidMandatoryAndOptionalFields